### PR TITLE
Added TeamCity

### LINKED ISSUE #4136: Icon Request: TeamCity
**Name:** TeamCity
**Website:** https://www.jetbrains.com/teamcity/
**Alexa rank:** 2121 (as of 27 Nov 2020)

**Official resources for icon and color:**

### PR DESCRIPTION
**Pull request by redrum-llik**
teamcity (1)

Close #4136
**Alexa rank:  2121 (as of 27 Nov 2020)**

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
**Source**: https://www.jetbrains.com/company/brand/logos/
I am TeamCity support engineer; this has been discussed internally and I will gladly assist with any questions.

### CODE REVIEW

**Comment by creepertron95** [comment]
This doesn't look very consistent with the other Jetbrains icons on the site compared with the current [Jetbrains](https://simpleicons.org/icons/jetbrains.svg) and [PyCharm](https://simpleicons.org/icons/pycharm.svg) icons

**Comment by redrum-llik** [comment]
> This doesn't look very consistent with the other JetBrains icons on the site compared with the current [JetBrains](https://simpleicons.org/icons/jetbrains.svg), [PyCharm](https://simpleicons.org/icons/pycharm.svg), [IntelliJ IDEA](https://simpleicons.org/icons/intellijidea.svg) icons.

Hello! Thank you, this is a good point - I have amended the base color and the icon itself, but the font size is still different (the icons you have referenced, like Intellij IDEA one added in PR #1497, were apparently made manually from a raster image whilst this one is based on an official SVG logo). If you feel like the larger font size is more readable, please let me know and I will fix accordingly.

File: icons/teamcity.svg
Comment (fbernhart) [inline icons/teamcity.svg:1]: `role="img"` is missing as attribute in the `<svg>` tag.

File: _data/simple-icons.json
Comment (fbernhart) [inline _data/simple-icons.json:6266]: In my opinion it would be better to use the branding page https://www.jetbrains.com/company/brand/logos as source.

Maybe one of the simple-icons maintainers can help here out.

**Comment by service-paradis** [comment]
Hello @redrum-llik , thanks for your contribution!

As you work for JetBrains, maybe you can help us out understand how we should use their logos. When opening Zip files for products at https://www.jetbrains.com/company/brand/logos/, we're given 2 SVGs.
1. `icon-[productName].svg`
2. `logo-[productName].svg`

By looking at the previous JetBrains icons we have, it looks like we used the former (as discussed in #3367). As @fbernhart said, help from another @simple-icons/maintainers would be appreciated too (I'm a newcomer!).

**Comment by runxel** [comment]
Hi y'all,
thanks to @redrum-llik for this PR! Always great to have this kind of official support!

I remember a debate with @PeterShaggyNoble where he advocated that we would need to use the logos _with_ the beam. But for the life of me I can't find it right now.
Since this PR doesn't has the beams/background stuff I think it's safe to assume that they are indeed _not_ needed.

> In my opinion it would be better to use the branding page jetbrains.com/company/brand/logos as source.
I agree with @fbernhart.

File: _data/simple-icons.json
Comment (ericcornelissen) [reply _data/simple-icons.json:6266]: If that is where the SVG and guidelines for TeamCity can be found, then I agree :+1:

**Comment by ericcornelissen** [comment]
To me this contribution seems to be the same general style as our existing JetBrains icons :thinking: If the font size is slightly different between the various icons that is fine from our perspective, provided that that is also the case with the official logos. If that is not the case with the official logos than some/all of the icons need to be updated. Mind you, I did not review this PR, but given that this one is based on an official SVG and the existing ones are based on raster images(*), I'd imagine this PR is correct and the others are (potentially) incorrect. On thing that alludes to this is the rough edges in [our existing JetBrains SVG](https://simpleicons.org/icons/jetbrains.svg)...


(*) based on @redrum-llik comment

**Comment by service-paradis** [comment]
Here is the Zip files found at https://www.jetbrains.com/company/brand/logos/
[teamcity_logos.zip](https://github.com/simple-icons/simple-icons/files/5625486/teamcity_logos.zip)
In this file, there are 2 versions for the logo/icon:
1. `icon-teamcity.svg` that looks like
icon-teamcity
2. `logo-teamcity.svg` that looks like
logo-teamcity

From what I understand, we based the other logos on `icon-teamcity.svg` while this PR is based on `logo-teamcity.svg`.

**Comment by ericcornelissen** [comment]
Ah, thanks for the clarification @service-paradis, now I get what the difference is :sweat_smile: Hmm, that is interesting. In that case I would like to know what @redrum-llik has to say about this before we continue.

**Comment by redrum-llik** [comment]
Hi all,

Thank you for the comments, much appreciated! Indeed, looks like the icons were either based on icon-*.svg (or similarly-styled raster images). I have reached out internally to discuss which ones would be preferable and will circle back shortly.

**Comment by redrum-llik** [comment]
Hello!

I have confirmed internally that the icons currently available in the simple-icons are preferred (e.g. based on icon-<product name>.svg from a corresponding brand logo pack). Therefore, the TeamCity icon has been updated to match the rest of JetBrains product icons available here:

teamcity (1)

According to the comments above:
1)  tag has been added;
2) source property has been updated to link to the https://www.jetbrains.com/company/brand/logos/ page;
3) beams present on "raw" icon-teamcity.svg (and other JB product icons) are indeed not required - in fact, this PR was started because of the request from customer who wanted to use the icon with GitHub badge, and beams would decrease readability. 

Hope this looks better now; please let me know if there is anything else I could look up or correct.

File: _data/simple-icons.json
Comment (ericcornelissen) [inline _data/simple-icons.json:6266]: A small formatting issue that should be fixed before this can be merged

```suggestion
        },
        {
            "title": "TeamCity",
            "hex": "000000",
            "source": "https://www.jetbrains.com/company/brand/logos/"
```

Comment (redrum-llik) [reply _data/simple-icons.json]: Done!

**Comment by fbernhart** [comment]
LGTM! 🎉

**Review by ericcornelissen — APPROVED** [review]

**Pull request merged**
